src/content/docs/integrations/prometheus-integrations/install-configure/remote-write-drop-data.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Drop data using Prometheus remote write 
+tags:
+  - Integrations
+  - Prometheus integrations
+  - Install and configure remote write
+translate:
+  - jp
+---
+
+You can drop data you aren't interested in keeping by changing the `remote_write` section of the YAML config file.
+
+<Callout variant="tip">
+You can also drop remote write data using NerdGraph. For more information, see [Drop data using NerdGraph](/docs/accounts/accounts/data-management/drop-data-using-nerdgraph/).
+</Callout>
+
+## Drop entire metric data points from remote write integration [#drop-entire]
+
+If a target is sending a noisy metric that you don't want sent to New Relic, you can specify that New Relic should drop that data. 
+
+### Example 
+
+Let's say you don't want to recieve data for the metric `node_memory_active_bytes` from an instance running at `localhost:9100`. Using the `write_relabel_config` entry shown below, you can target the metric name using the `__name__` label in combination with the instance name.
+
+```
+remote_write:
+- url: https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=macbook-server-cluster
+  bearer_token: <redacted>
+  write_relabel_configs:
+    - source_labels: ['__name__', 'instance']
+      regex: 'node_memory_active_bytes;localhost:9100'
+      action: 'drop'
+```
+
+This tells Prometheus that you want to do some action against metrics with these labels. To limit which metrics with these labels are affected you must include some value for regex. By default this value is set to `.*` so will include all metrics. In this case, it will drop all metric data points coming out of Prometheus via remote write. 
```

Comment:
Line 35: "To limit which metrics with these labels are affected`,` you must include some value for regex."